backend/pricing/pricing.go

```diff
@@ -298,6 +298,23 @@ var ProductPrices = map[backend.PlanType]map[model.PricingProductType]ProductPri
 	},
 }
 
+func GetSessionsUsageHistory(ctx context.Context, DB *gorm.DB, ccClient *clickhouse.Client, workspace *model.Workspace, dateRange backend.DateRangeRequiredInput) (result []int64, err error) {
```

Comment:
are you using this or just querying clickhouse?

Reply:
good catch